sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ListBatchesResponse.java

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * A list of paginated Batch objects.
+ */
+@Immutable
+public final class ListBatchesResponse implements JsonSerializable<ListBatchesResponse> {
```

Comment:
I deleted this File locally and regenerated SDK and no failure. So, this class is not needed anymore. 

Probably we don't need it as it is same set of fields as the `PageableList<T>`

Can you double check it and see if we need it?

Reply:
This was renamed to use the the `OpenAIPageableListOf<Batch>`. I think you can still find it in `implementation.models.OpenAIPageableListOfBatch`